image_processing/src/app.py

```diff
@@ -0,0 +1,100 @@
+from h2o_wave import Q, app, ui, main
+import ip_utils as ip
```

Comment:
It might be good to use relative imports here: `import .ip_utils as ip`

I think most users will run: `./venv/bin/wave run src.app` vs. `../venv/bin/wave run app` from within the `src` folder. Or document where and how to run in the README